content/pages/privacy-policy.md

```diff
@@ -0,0 +1,93 @@
+title: Mixxx Privacy Policy
+slug: privacy-policy
+
+## Introduction
+
+Mixxx is a non-profit project managed by volunteers.
+
+There is no business model around Mixxx, and the only revenue we receive is through donations.
+
+
+For privacy policies and conformance to GDPR, a few distinctions will be made between:
+
+1. Mixxx application
+2. Main Mixxx website
+3. Community Mixxx websites
+
+## 1. Mixxx Application
+
+The Mixxx application doesn't require or use any kind of user account, nor does it collect any user data when being used.
+
+
+Mixxx does not collect any data, nor any telemetry, when Mixxx is being run.

```

Comment:
This sentence is not ultimative correct, because of the mixxx log 
```suggestion
Mixxx does not perform any telemetry and does not share data without the express request of the user (Opt-in). 
```

Reply:
Were do we have Opt-In telemetry. If we have, this needs to explained in detail.

Reply:
If you refer to the mixxx.log we'd also need to mention the database which stores last played, date added etc. ;)
But I think that is covered by the paragraph about bug reporting, no?

Reply:
I combined both sentences now.